docs/database/database-spans.md

```diff
@@ -74,7 +74,7 @@ These attributes will usually be the same for all operations performed over the
 |---|---|---|---|---|---|
 | [`db.system`](../attributes-registry/db.md) | string | An identifier for the database management system (DBMS) product being used. See below for a list of well-known identifiers. | `other_sql` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`db.collection.name`](../attributes-registry/db.md) | string | The name of a collection (table, container) within the database. [1] | `public.users`; `customers` | `Conditionally Required` [2] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`db.name`](../attributes-registry/db.md) | string | This attribute is used to report the name of the database being accessed. For commands that switch the database, this should be set to the target database (even if the command fails). [3] | `customers`; `main` | `Conditionally Required` If applicable. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.collection.namespace`](../attributes-registry/db.md) | string | The namespace containing database objects, fully qualified within the server address and port. [3] | `customers`; `test.users` | `Conditionally Required` If applicable. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
How should the implementation handle cases when a query touches multiple tables across multiple schemas?

MS SQL example with 2 schemas (`schema1` and `schema2`):

```
SELECT schema1.table1.C2
FROM schema1.table1
INNER JOIN schema2.table2 ON schema1.table1.C1 = schema2.table2.C1
```

1) To the "where should the schema be?" discussion: regardless of what we settle on, I think the spec should say something about queries with multiple schemas.
2) Not related to the PR, but shouldn't the `db.collection.name` say something about such cases as well? (specifically queries with multiple tables)

I think both multiple tables and multiple schemas can be handled the same (either choose 1 (e.g. from the SELECT clause). or comma separate, or something else, but do the same for `db.collection.namespace` and `db.collection.name`).

Reply:
> Not related to the PR, but shouldn't the `db.collection.name` say something about such cases as well? (specifically queries with multiple tables)

I believe it does, check out the footnote for `db.collection.name` in section https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-spans.md#common-attributes

> [2]: If readily available. Otherwise, if the instrumentation library parses `db.query.text` to capture `db.collection.name`, then it SHOULD be the first collection name found in the query.

did we maybe miss this somewhere else?

Reply:
Oh, indeed, sorry (I'd have expected that part under footnote [1] - but maybe it's just me, doesn't matter). 

So cases with multiple tables seem to be handled fine. So if we decide to capture schema, I suggest we have something similar for that as well.